db/migrate/20160720085321_create_container_template.rb

```diff
@@ -0,0 +1,27 @@
+class CreateContainerTemplate < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_templates do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :ems_created_on
+      t.string     :resource_version
+      t.belongs_to :ems, :type => :bigint
+      t.belongs_to :container_project, :type => :bigint
+      t.text       :objects
+      t.timestamps
+    end
+
+    create_table :container_template_parameters do |t|
+      t.string     :name
+      t.timestamp  :ems_created_on
+      t.string     :display_name
+      t.string     :description
+      t.string     :value
+      t.string     :generate
```

Comment:
Generate specifies the generator to be used by Openshift to generate random string from an input value specified by From field. It can be set for example to 'expression' for generating the value using a pseudo regular expression syntax.